track/track.py

```diff
@@ -237,6 +237,23 @@ def get_overlap_ratio(self, other_track, threshold=0.05):
 
         return frames_overlapped / len(self)
 
+    def get_track_frame(self, frame, track_frame_number):
+        bounds = self.bounds_history[track_frame_number]
+        thermal = bounds.subimage(frame[0])
+        filtered = bounds.subimage(frame[1])
+        mask = bounds.subimage(frame[2])
+        flow = bounds.subimage(frame[3])
+        flow2 = bounds.subimage(frame[4])
+
+        # make sure only our pixels are included in the mask.
+        mask[mask != bounds.id] = 0
+        mask[mask > 0] = 1
+
+        # stack together into a numpy array.
+        # by using int16 we loose a little precision on the filtered frames, but not much (only 1 bit)
```

Comment:
I wonder how important that bit is :)

Reply:
This was already there, but does seem pointless to me since we have already kept float32 up until this point